main.py

```diff
@@ -60,18 +73,51 @@ def read_files(args):
             director=line[5],
             rating=line[6]
         )
-        # Divide actors(line[3]=actors) incording to "," and add into actor_list
-        movie.actors = Set(line[3].split(', '))
+        #Add movie and into movies set in dataset
         data.movies.add(movie)
+        #Send informations of every director to Director class in dataset
+    for director in director_set:
+        #Create director
+        director=dataset.Director(
+            ID=director_id,
+            name=director
+        )
+        director_id+=1
+        # Add director into directors set in dataset
+        data.directors.add(director)
+
+
+    #Send every actor to Actor class in dataset
+    for actor in actor_set:

```

Comment:
Bu kısım da actor_set yerine direk okunduğu yerde olmalıydı.
Class tanımlamamızın nedeni bu local dict ve setlerden kurtulmak ve daha anlaşılır bir kod yazmak olduğu için bunlara dikkat edelim.